### PR TITLE
LPS-167017 Add button role to control menu anchors

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-reports-web/src/main/resources/com/liferay/layout/reports/web/internal/product/navigation/control/menu/icon.tmpl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <li class="control-menu-nav-item d-none d-sm-inline-flex layout-reports-icon ${cssClass}">
-	<a aria-label="${title}" class="control-menu-icon lfr-portal-tooltip product-menu-toggle sidenav-toggler" data-content="body" data-open-class="lfr-has-layout-reports-panel open-admin-panel" data-target="#${portletNamespace}layoutReportsPanelId" data-title="${title}" data-toggle="liferay-sidenav" data-type="fixed-push" data-type-mobile="fixed" href="javascript:void(0);" id="${portletNamespace}layoutReportsPanelToggleId">
+	<a aria-label="${title}" class="control-menu-icon lfr-portal-tooltip product-menu-toggle sidenav-toggler eadadassad" data-content="body" data-open-class="lfr-has-layout-reports-panel open-admin-panel" data-target="#${portletNamespace}layoutReportsPanelId" data-title="${title}" data-toggle="liferay-sidenav" data-type="fixed-push" data-type-mobile="fixed" href="javascript:void(0);" id="${portletNamespace}layoutReportsPanelToggleId" role="button">
```
Comment (ealonso) [inline modules/apps/layout/layout-reports-web/src/main/resources/com/liferay/layout/reports/web/internal/product/navigation/control/menu/icon.tmpl:2]: @sandrodw3 nice class eadadassad

@victorg1991 it is better than WIO

Comment (sandrodw3) [reply modules/apps/layout/layout-reports-web/src/main/resources/com/liferay/layout/reports/web/internal/product/navigation/control/menu/icon.tmpl:2]: 🤣

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/control/menu/edit_layout_control_menu_entry_icon.tmpl [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <li class="control-menu-nav-item">
-	<a aria-label="${configurePage}" class="control-menu-icon lfr-portal-tooltip" data-qa-id="editLayout" data-title="${configurePage}" href="${editPageURL}">
+	<a aria-label="${configurePage}" class="control-menu-icon lfr-portal-tooltip" data-qa-id="editLayout" data-title="${configurePage}" href="${editPageURL}" role="button">
```
Comment (victorg1991) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/control/menu/edit_layout_control_menu_entry_icon.tmpl:2]: Can't we use a button? 😢

File: modules/apps/layout/layout-reports-web/src/main/resources/com/liferay/layout/reports/web/internal/product/navigation/control/menu/icon.tmpl (hunk 1 above)
Comment (victorg1991) [reply modules/apps/layout/layout-reports-web/src/main/resources/com/liferay/layout/reports/web/internal/product/navigation/control/menu/icon.tmpl]: JAJA

**Comment by victorg1991** [comment]
ci:forward

**Comment by ozysouza** [comment]
Just started reviewing :)

**Comment by ozysouza** [comment]
Please see https://github.com/liferay-echo/liferay-portal/pull/10360

**Pull request closed without merging**